C#Backend/InpatientTherapySchedulingProgramTests/InpatientTherapySchedulingProgramTests.csproj

```diff
@@ -24,4 +24,10 @@
     <ProjectReference Include="..\InpatientTherapySchedulingProgram\InpatientTherapySchedulingProgram.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="ServiceTests\HoursWorkedServiceTests.cs">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Compile>
+  </ItemGroup>
+
```

Comment:
Any idea what this does?

Reply:
Uh no, I have no clue why that was added. Do you want me to remove it